Frontend/ui-driver/src/Screens/HomeScreen/Handler.purs

```diff
@@ -138,3 +138,11 @@ homeScreen = do
     RefreshGoTo updatedState -> do
       modifyScreenState $ HomeScreenStateType (\_ → updatedState)
       App.BackT $ App.BackPoint <$> (pure $ REFRESH_GOTO updatedState)
+    EarningsScreen updatedState -> do 
+      modifyScreenState $ HomeScreenStateType (\_ -> updatedState)
+      App.BackT $ App.BackPoint <$> (pure $ HOMESCREEN_NAV GoToEarningsScreen)
+-- DTHS.GoToStart screenState -> do
```

Comment:
why is this commented? any use in future? if yes, please add comment as well with a note otherwise remove commented code

Reply:
Removed